plugins/app-rating/README.md

```diff
@@ -0,0 +1,43 @@
+# app-rating
+
+Welcome to the app-rating plugin!
+
+_This plugin was created through the Backstage CLI_
```

Comment:
Can we include a screen shot? It really helps people get a better impression of what the plugin does

Reply:
Absolutely